packages/terra-form-select/src/multiple/Frame.jsx

```diff
@@ -428,6 +423,20 @@ class Frame extends React.Component {
     }
   }
 
+  /**
+   * Handles the mouse down events.
+   * @param {event} event - The mouse down event.
+   */
+  handleMouseDown(event) {
+    // Preventing default events stops the search input from losing focus.
+    // The default variant has no search input therefore the mouse down gives the component focus.
+    event.preventDefault();
+    if (event.target.hasAttribute('data-terra-form-select-input')) {
+      this.input.focus();
+    }
+    this.openDropdown(event);
+  }
```

Comment:
This function is never called. The function being called in this file is the `static handleMouseDown` defined above the constructor. The Multiple and Tag variants were updated to open and close the dropdown on click events, rather than mouse down events.

https://github.com/cerner/terra-core/pull/3066/

Reply:
Removed